src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -47,9 +32,19 @@ export const TalkShowStudio: FC<TalkShowStudioProps> = ({ venue }) => {
   const stage = useStage();
   const { userId, profile } = useUser();
   const currentVenue = useSelector(currentVenueSelectorData);
-  const remoteUsers = useAgoraRemotes({ client: remotesClient });
   const isRequestToJoinStageEnabled = venue.requestToJoinStage;
 
+  const [remotesClient, screenClient, cameraClient] = React.useMemo(() => {
```

Comment:
```suggestion
  const [remotesClient, screenClient, cameraClient] = useMemo(() => {
```

Reply:
@0xdevalias fixed in efa8913141c5d517382f8f25178d4967c902a3df